year-pages/2019-conferences-list.md

```diff
@@ -2,6 +2,7 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [React India](https://www.reactindia.io/) | 23-25 Aug | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native & GraphQL. This edition will be gathering Front-end and Full-stack developers across the globe in the cool place of India. In this single track event you will learn more about React and surrounding topics while meeting some of the leading talents around the globe. Regular and lightning talks will cover various React.JS and FE topics including React Native, GraphQL, VR & AR, Redux, Preact and more! | No |
```

Comment:
1. The date on the website seems to be 22-24 August. Can you update the table to reflect this?

2. Can you also update the description to be 2-3 lines at max (like other conferences in the table)?

Reply:
1. Ah! Looks like they have changed the dates.

2. Done